### PR TITLE
Merge FBSSDC repo into binjs-ref

### PR DESCRIPTION
**Pull request by vdjeric**
Note that FBSSDC commit history is preserved and visible via "git log --follow"

I'll do follow-up commits to integrate Travis and other niceties

@Yoric @dominiccooney

### CODE REVIEW

File: fbssdc/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+language: python
```
Comment (RReverser) [inline fbssdc/.travis.yml]: I suppose this file will have to be modified somehow or manually called from top-level .travis.yml?

File: fbssdc/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Contributing to FBSSDC
```
Comment (RReverser) [inline fbssdc/CONTRIBUTING.md]: Is this and other similar files necessary given that it's now just a folder and not a separate repo?

File: fbssdc/.travis.yml (hunk 1 above)
Comment (vdjeric) [reply fbssdc/.travis.yml]: Yes I will figure out how to get the Travis files to play together

File: fbssdc/CONTRIBUTING.md (hunk 2 above)
Comment (vdjeric) [reply fbssdc/CONTRIBUTING.md]: It's not necessary, I'll remove such files after the merge

File: fbssdc/.travis.yml (hunk 1 above)
Comment (Yoric) [reply fbssdc/.travis.yml]: It would be nice to get this done before the merge.

**Comment by Yoric** [comment]
Did you update this, @vdjeric ?

**Comment by vdjeric** [comment]
Combined .travis.yml's

**Comment by vdjeric** [comment]
Let's land it?

**Pull request merged**